### DISCUSSION TITLE
agnostic vite-plugin-devtools foundation

### DISCUSSION THREAD

**Discussion by birkskyum** [thetarnav/solid-devtools#283] [General]
Would it make sense for this project to leverage a foundation shared by the vue / nuxt devtools?

https://github.com/pheno-agency/vite-plugin-devtools

**Comment by thetarnav** [comment]
Thanks for linking this. I'm not planning on adding a dependency, but I'm interested in what ideas can be taken from this. Do I understand it correctly that it setups a way for the vite plugin and the website to communicate? I haven't tried doing that so that's very valuable. I'm not sure what I would use that for yet though. If you have some ideas please let me know :)

**Reply by birkskyum** [reply]
I actually don't know, I've just been very impressed by the track-record of agnostic projects coming out of the vue community, like vite, nitro and volar
